app/src/pages/admin/allUsersPage/allUsersGrid/nameColumn/nameColumn.jsx

```diff
@@ -96,7 +102,16 @@ export class NameColumn extends Component {
             />
           </div>
         )}
-        <span className={cx('name')}>{value.fullName}</span>
+        <span
+          className={cx('name')}
+          title={
+            value.fullName && value.fullName.length > maxNameLengthForTitle
```

Comment:
Maybe it's better to show the tooltip in every case? Could you please clarify this?